authentication-service/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use the official Golang image as the base image
+FROM golang:1.23-alpine
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy the Go module files
+COPY go.mod go.sum ./
+
+# Download the Go module dependencies
+RUN go mod download
+
+# Copy the source code
+COPY . .
+
```

Comment:
Hi, the dockerfile looks great! I was just wondering how does the docker container get the env variables to retrieve from the database if it is not specified here?